src/components/SlateEditor/plugins/textTransform/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export default function paragraphPlugin() {
```

Comment:
```suggestion
export default function textTransformPlugin() {
```
Denne skal vel hete `textTransformPlugin`?

Reply:
Haha. Lett å glømme seg bort ved copy paste.